functionnal_tests/cypress/integration/account/content_account_spec.js

```diff
@@ -2,6 +2,8 @@ import { PAGES } from '../../support/urls_commands'
 import { SELECTORS as s } from '../../support/generic_selector_commands'
 import baseUser from '../../fixtures/baseUser.json'
 
+
+
```

Comment:
This is probably too many lines (would not pass `standard`'s linting, but we do not currently lint Cypress files).

The following suggestion would fix it but feel free to ignore, we will probably clean up all the linting issues in cypress files when we are ready to do it.

```suggestion


```